### DISCUSSION TITLE
Transfer 06 - Wrong result in Step 11

### DISCUSSION THREAD

**Discussion by joelweihrauch** [eclipse-edc/Samples#89] [Q&A]
Hi,

currently i am working in my project with a use case equivalent to [transfer-06-consumer-pull-http](https://github.com/eclipse-edc/Samples/tree/main/transfer/transfer-06-consumer-pull-http) 
Since the update on 0.1.3 i experience a problem in the consumer pull sample. 
I follow the steps in the README(Not changing anything) and get back other results as described in the step 11(https://github.com/eclipse-edc/Samples/blob/main/transfer/transfer-06-consumer-pull-http/README.md#11-pull-the-data). 

Following my result:

```json 
{"id":"44a59057-f799-4854-8a26-0830f76825d3",
"endpoint":"http://localhost:19291/public/",
"authKey":"Authorization",
"authCode":"eyJhbGciOiJSUzI1NiJ9.eyJleHAiOjE2ODk1ODA5OTksImRhZCI6IntcInByb3BlcnRpZXNcIjp7XCJodHRwczovL3czaWQub3JnL2VkYy92MC4wLjEvbnMvdHlwZVwiOlwiSHR0cERhdGFcIixcImh0dHBzOi8vdzNpZC5vcmcvZWRjL3YwLjAuMS9ucy9uYW1lXCI6XCJUZXN0IGFzc2V0XCIsXCJodHRwczovL3czaWQub3JnL2VkYy92MC4wLjEvbnMvYmFzZVVybFwiOlwiaHR0cHM6Ly9qc29ucGxhY2Vob2xkZXIudHlwaWNvZGUuY29tL3VzZXJzXCJ9fSIsImNpZCI6Ik9RPT06WVhOelpYUkpaQT09Ok9XWmpNMkV5WmpBdE16UmlZaTAwWm1aaExXRXdZMlV0WkRFd09HRTVOV1k0T1RZNSJ9.ledo9PGvWy-REzd-ysTdR_1oitcm9AVPxuipGJnR2wifyQ6Xb5OeOz-5VaIXiHlif9YrL0WnqTlRhBVf-9MHtzXDKzBzdwzEGAIY1z_s2XvtaQDo7SRN_V3uzcxu6QrKCeKeq0hiefj3lRwrSqW1KXBVF-YhgOwYciRJE14ECTXz8yxxRaIQ9izjchYSc9WE-1O-vD6vddv1mJlNgcAALUcL9mIztAj3TavxMJtqh1jIT-d7W-VO9qM3g2yfBNZwKpAK3qIEab_rdHYc-XdlJX_B__nq8VE896iDTzqi8w7_r_JpnLeT5lxKJ37AdAz29AMXP5QWfoYDg4raFvYW1Q",
"properties":{}}
```

Do i have to configure something else? How can i achieve the result described in the readme?

Thank you

**Comment by johnnychoque** [comment]
Hi @joelweihrauch,

I had not seen this thread before. I just opened a discussion related to your question 
[https://github.com/eclipse-edc/Samples/discussions/113](https://github.com/eclipse-edc/Samples/discussions/113)

Although the endpoint port is different from the one you get, it works correctly with both by running the curl command in step 11.

**Comment by joelweihrauch** [comment]
Hi @johnnychoque,

since i had digged into the code and discussed it in Q&A session to resolve this "problem", i figured out that the "Pull-Flow" has been changed/simplized. This is knowledge from current state in version 0.1.3 so i can't say something about the following versions. 
The "Pull-Flow" is now processed in step 11 directly on the Provider-Control-Plane so the Consumer-Control-Plane is not between this request anymore.

Please correct me if i am misleading here. @ndr-brt @ronjaquensel
